container/go/cmd/puller/puller.go

```diff
@@ -120,6 +122,56 @@ func pull(imgName, dstPath, format, cachePath string, platform v1.Platform) erro
 		}
 	}
 
+	if err := generateSym(img, dstPath); err != nil {
+		return errors.Wrapf(err, "failed to generate symbolic links to pulled image at %s", dstPath)
+	}
+
+	return nil
+}
+
+// generateSym creates symbolic links to the config.json and .tar.gz layers
+// for use with container_import rule.
+func generateSym(img v1.Image, dstPath string) error {
+	targetDir := path.Join(dstPath, "blobs/sha256")
+	symlinkDir := path.Join(filepath.Dir(dstPath), "image")
+
+	// symlink for config.json.
```

Comment:
Elaborate this comment explaining the OCI layout. i.e., the config file is expected to exist under blobs/sha256 as a file named the same as the sha256 digest of the config.